srv/modules/runners/config.py

```diff
@@ -216,38 +229,32 @@ def write(self):
         return False
 
 
-def deploy(*args):
+def deploy_salt_conf(*args):
     ''' Creates Salt configuration '''
-
-    if args and args[0] == "roles":
-        deploy_roles()
-    elif args and args[0] == "global":
-        deploy_global()
-    else:
-        deploy_roles()
-        deploy_global()
-    return ""
+    if all([deploy_roles(), deploy_global()]):
```

Comment:
I have one place where I only use deploy_roles for example.

Maybe we can have an overarching `deploy_conf_all` which calls `deploy_salt_conf` and `deploy_ceph_conf`.

On the other hand I don't know _when_ to use that, since those can't be executed together.

Reply:
While your suggestion sounds a bit more consistent, it's probably not really describing what it is  doing.

`deploy_deepsea_all` sounds to me like it'd be deploying all of deepsea (config related)

Maybe we should be more verbose in our function names.

`deploy_deepsea_configuration`, which consists of roles and global

`deploy_deepsea_configuration_for_roles`
`deploy_deepsea_configration_for_global_yml`

I'm not convinced that this is the right direction at all :)

`deploy_ceph_configuration` Would create and distribute the ceph.conf. I think this is relatively straight forward